.github/workflows/update-rpm-repo.yml

```
@@ -1,436 +1,350 @@
+            exit 1
+          fi
+          
+          echo "version=$VERSION" >> $GITHUB_OUTPUT
+          echo "Final version: $VERSION"
 
-      - name: Wait for COPR build completion (for tags)
-        if: ${{ github.ref_type == 'tag' }}
+      - name: Wait for COPR build completion
+        if: ${{ github.event_name == 'release' || (github.event_name == 'workflow_dispatch' && github.event.inputs.wait_for_copr == 'true') }}
+        working-directory: flightctl
         run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2162:info:15:12: read without -r will mangle backslashes

(shellcheck)